packages/react/src/HyperlinkToolbar/HyperlinkToolbarFactory.tsx

```diff
@@ -1,61 +1,19 @@
-import { createRoot, Root } from "react-dom/client";
 import {
   HyperlinkToolbar,
   HyperlinkToolbarDynamicParams,
   HyperlinkToolbarFactory,
   HyperlinkToolbarStaticParams,
 } from "@blocknote/core";
-import { MantineProvider } from "@mantine/core";
-import Tippy from "@tippyjs/react";
 import { HyperlinkToolbar as ReactHyperlinkToolbar } from "./components/HyperlinkToolbar";
-import { BlockNoteTheme } from "../BlockNoteTheme";
-// import rootStyles from "../../../core/src/root.module.css";
+import { ElementFactory as ReactElementFactory } from "../ElementFactory";
```

Comment:
Why not just rename it ReactElementFactory? Then you don't need `ElementFactory as ReactElementFactory` all the time anymore